gemato/openpgp.py

```diff
@@ -155,6 +155,10 @@ def _spawn_gpg(self, argv, stdin='', env_override={}):
         env['TZ'] = 'UTC'
         env.update(env_override)
 
+        # Allow to override default 'gpg' client
+        if env.get('GNUPG', None):
```

Comment:
I'm pretty sure this will break that one call site where we call `gpgconf`.

Reply:
Hum indeed. Any suggestion?

Reply:
Maybe we could check for what would be set `GNUPGCONF`, standing for `gpgconfg` and replace depending of the `argv[0]` call `gpg/gpgconf`?

Reply:
To be honest, I don't really like the way it's done now anyway but I'm yet to think of a good alternative. Maybe we could declare `GNUPG` and `GNUPGCONF` as globals, and use these in the calls.

Reply:
Yes that would be much better. Originally I tried to not being to invasive into the code. I'll adapt with respect to your suggestion.